streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -18,8 +18,8 @@
 package org.apache.spark.streaming
 
 import java.io._
-import java.util.concurrent.Executors
-import java.util.concurrent.RejectedExecutionException
+import java.util.concurrent.{ArrayBlockingQueue, RejectedExecutionException,
```

Comment:
do we usually break up the import lines this way? i dont think so, but I may be wrong.